README.md

```diff
@@ -34,28 +37,24 @@ CyberBear is part of **HackToFuture-2024**, under the Cyber Security domain (Pro
 - **Educational Institutions**: Interactive modules integrate easily into curricula, enabling educators to provide hands-on cybersecurity education for students.
 - **Businesses**: CyberBear offers tailored training programs that educate employees on cybersecurity best practices, bolstering organizational security.
 
+![Educational Modules](https://media.giphy.com/media/l3vR3S8rL6rVSHgSY/giphy.gif)
+
+
 ## Setup and Installation
 1. **Clone the Repository**:
     ```bash
-    git clone https://github.com/yourusername/CyberBear.github.io.git
-    cd CyberBear.github.io
+   git clone https://github.com/Vinay-Khanagavi/Cyberbear.github.io.git
     ```
+## Join Our Community
+Join our Discord community to stay updated, get support, and connect with other users and contributors:
 
-## Contribution Guidelines
-We welcome contributions to enhance CyberBear! Here’s how you can get started:
-
-1. Fork the repository.
-2. Create a new branch for your feature or bug fix.
-3. Submit a pull request with a clear description of your changes.
+[![Join Our Discord](https://img.shields.io/badge/Discord-Join%20Our%20Community-7289DA?style=for-the-badge&logo=discord)](https://discord.gg/ZA7VSKHb9r)
 
-### Development Requirements
-- **HTML & CSS**
-- **JavaScript**
-- **Python** (latest stable version)
-- Any additional packages as noted in `requirements.txt`
+![Community](https://media.giphy.com/media/3o7TKFv9t9O4af7Tzq/giphy.gif)
 
 ## License
 This project is licensed under the MIT License. See the [LICENSE](LICENSE) file for more details.
 
-![CyberBear](https://github.com/Vinay-Khanagavi/CyberFit.github.io/assets/116386393/20f86349-0d7c-4db1-a9bb-7c34d7b3f206)
+![CyberBear Banner](https://github.com/Vinay-Khanagavi/CyberFit.github.io/assets/116386393/20f86349-0d7c-4db1-a9bb-7c34d7b3f206)
+
```

Comment:
Can you change the demo pics to the updated UI. U can get them from
https://vinay-khanagavi.github.io/Cyberbear.github.io/index.html

Reply:
Ok, I'll change it.